services/api-server/src/simcore_service_api_server/models/schemas/meta.py

```diff
@@ -9,8 +9,12 @@ class Meta(BaseModel):
     released: dict[str, VersionStr] | None = Field(
         None, description="Maps every route's path tag with a released version"
     )
-    docs_url: AnyHttpUrl = Field(default="https://docs.osparc.io")
-    docs_dev_url: AnyHttpUrl = Field(default="https://api.osparc.io/dev/docs")
+    docs_url: AnyHttpUrl = Field(
```

Comment:
I see now what happened ... All these mypy errors happened after the upgrade. I am already taking care of thse changes in https://github.com/ITISFoundation/osparc-simcore/pull/4118

Please drop all the changes related to mypy in this PR